core/test/docs.toml

```
@@ -14,9 +14,9 @@ results_dir = "results" # required
 time = "basin/time.arrow"
 
 [allocation]
-timestep = 86400
-use_allocation = true
-objective_type = "quadratic_relative"
+timestep = 86400                   # optional (required if use_allocation = true), default 86400
+use_allocation = true              # optional, default true
```

```suggestion
use_allocation = false             # optional, default true
```

Default is false. I think requiring modelers to opt-in still makes sense, so they can focus on getting a correct model without allocation first.